app/templates/serviceLearning/slcProposal.html

```diff
@@ -1,4 +1,11 @@
-<h2 class="text-center pt-4 pb-2"> New Proposal For Service-Learning Course</h2>
+ <h2 class="text-center pt-4 pb-2">Proposal For Service-Learning Course</h2>
+
+      <h3 class="text-center pb-4">Proposal Status:
+        <span style="color: rgb(12, 8, 245);">
+                {{ courseStatus.status }}
+            </span>
```

Comment:
We don't need a color unless it is going to change for different statuses

Reply:
removed